querydsl-sql-codegen/src/main/java/com/mysema/query/sql/codegen/MetaDataExporter.java

```diff
@@ -109,6 +109,9 @@
     private boolean exportForeignKeys = true;
 
     private boolean spatial = false;
+    
+    @Nullable
+    private String tableTypesToExport; 
```

Comment:
If you don't want to tokenize the String, you can also use [this way to automatically receive a `List<String>`](http://maven.apache.org/guides/plugin/guide-java-plugin-development.html#Collections)
I don't know if specifying only one value (`<tableTypesToExport>TABLE</tableTypesToExport>`) would work after that, but if it does, then I think this would improve readability of the pom as well.